x-pack/plugin/esql/qa/testFixtures/src/main/resources/data/mv_text.csv

```diff
@@ -0,0 +1,4 @@
+@timestamp:date         ,message:text
+2023-10-23T13:55:01.543Z,[Connected to 10.1.0.1, Banana]
+2023-10-23T13:55:01.544Z,Connected to 10.1.0.1
+2023-10-23T13:55:01.545Z,[Connected to 10.1.0.1, More than one hundred characters long so it isn't indexed by the sub keyword field with ignore_above:100]
```

Comment:
What about adding also a single-value over `ignore_above`? So we have all the cases here

Reply:
Yeah. I should do that.